src/util/net/util.h

```diff
@@ -0,0 +1,50 @@
+#ifndef HEADER_fd_src_util_net_util_h
+#define HEADER_fd_src_util_net_util_h
+
+#include "fd_udp.h"
+#include "fd_eth.h"
+
+typedef struct __attribute__((packed)) {
+  fd_eth_hdr_t eth[1];
+  fd_ip4_hdr_t ip4[1];
+  fd_udp_hdr_t udp[1];
+} fd_net_hdrs_t;
+
+FD_PROTOTYPES_BEGIN
+
+/**
```

Comment:
Just because this is util code can you match the formatting for the comment, eg:

```
/* Helper method to ....
    and other text... */

fd_net_hdrs_t *
```